libr/debug/dreg.c

```diff
@@ -82,7 +82,8 @@ R_API int r_debug_reg_sync(RDebug *dbg, int type, int write) {
 	return true;
 }
 
-R_API int r_debug_reg_list(RDebug *dbg, int type, int size, int rad, const char *use_color) {
+R_API bool r_debug_reg_list(RDebug *dbg, int type, int size, int rad, const char *use_color) {
+	size_t i;
```

Comment:
move this inside the `if`.

Reply:
Thanks for the reminder. I am sorry that I resolved those requests before actually committing the changes as I was keeping track of them here.